build.gradle

```diff
@@ -8,14 +8,17 @@ buildscript {
     dependencies {
         classpath "gradle.plugin.com.github.spotbugs:gradlePlugin:1.6.0"
     }
+    dependencies {
+        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.0'
+    }
 }
 
 plugins {
     id 'application'
     id 'io.spring.dependency-management' version '1.0.3.RELEASE'
     id 'org.springframework.boot' version '1.5.6.RELEASE'
-    id 'org.owasp.dependencycheck' version '1.4.5.1'
-    id 'com.github.ben-manes.versions' version '0.15.0'
+    id 'org.owasp.dependencycheck' version '2.1.0'
+    id 'com.github.ben-manes.versions' version '0.17.0'
```

Comment:
just saw this while checking the latest version looks useful:
https://github.com/patrikerdes/gradle-use-latest-versions-plugin